fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/data/LAORepository.java

```diff
@@ -308,6 +313,27 @@ private boolean handleStateLao(String channel, StateLao stateLao) {
     return false;
   }
 
+
+  private boolean handleElectionSetup(String channel, ElectionSetup electionSetup) {
+    Lao lao = laoById.get(channel).getLao();
+    Log.d(TAG, "handleElectionSetup: " + channel + " name " + electionSetup.getName());
+
+    Election election = new Election();
+    election.setId(electionSetup.getId());
+    election.setName(electionSetup.getName());
+    election.setCreation(electionSetup.getCreation());
+
+    election.setStart(electionSetup.getStartTime());
+    election.setQuestion(electionSetup.getQuestions().get(0).getQuestion());
```

Comment:
It could be better for robustness to check if the questions list is non empty before accessing the first one